mix/qml/js/ProjectModel.js

```diff
@@ -86,6 +85,7 @@ function saveProjectFile()
 		for (var i = 0; i < projectListModel.count; i++)
 			projectData.files.push(projectListModel.get(i).fileName);
 
+		projectSaving(projectData);
```

Comment:
SaveProjectFile is used to save the .mix file only, so I think projectSaving should stay in SaveProject.
But the name SaveProjectFile is confusing, maybe better use SaveProjectDesc